src/Model/GroundWaterFlow/submodules/sm-gwf-sfr-steady.f90

```diff
@@ -0,0 +1,305 @@
+submodule(SfrModule) sm_gwf_sfr_steady
```

Comment:
Just throwing this out there for discussion -- if the submodule is part of SfrModule, then does the submodule name need to include "gwf_sfr" in it?  (as a separate note, we probably need to rename SfrModule to GwfSfrModule at some point to be consistent with other parts of the code).

In following with our other naming conventions, should the submodule name, then be SteadySubmodule since it by definition determined to part of gwf_sfr?